apprentice-react-o-shop/src/components/NavBar/NavBar.js

```diff
@@ -16,24 +19,16 @@ function NavBar(){
                 <li className="nav-item active">
```

Comment:
small challenge: right now when you select one of the other pages the home link still remains active in the navbar. Usually we would want whatever active page the user is viewing to be the highlighted link in the navbar.